modules/notification/ui/ui.go

```diff
@@ -51,35 +58,39 @@ func (ns *notificationService) NotifyCreateIssueComment(doer *models.User, repo
 	if comment != nil {
 		opts.commentID = comment.ID
 	}
-	ns.issueQueue <- opts
+	_ = ns.issueQueue.Push(opts)
 }
 
 func (ns *notificationService) NotifyNewIssue(issue *models.Issue) {
-	ns.issueQueue <- issueNotificationOpts{
+	_ = ns.issueQueue.Push(issueNotificationOpts{
 		issueID:              issue.ID,
 		notificationAuthorID: issue.Poster.ID,
-	}
+	})
 }
 
 func (ns *notificationService) NotifyIssueChangeStatus(doer *models.User, issue *models.Issue, actionComment *models.Comment, isClosed bool) {
```

Comment:
Not related to this PR, but shouldn't at least the actionComment ID be passed through as well?

Reply:
I would defer to @lunny on this. I don't really understand the notification service.

Reply:
@guillep2k `actionComment *models.Comment` we can get the ID from the `actionComment`.